### ISSUE TITLE
Fatal swiftlint error in SwiftyStoreKit-iOS-Demo and SwiftyStoreKit-macOS-Demo

### ISSUE THREAD

**Issue by mikesax** [bizz84/SwiftyStoreKit#273]
Platform: iOS, macOS
Environment: Sandbox
Version: bizz84/SwiftyStoreKit 9d1b446
Using swiftlint 0.22.0 and Xcode 9.0 (9A235)

Build using Carthage fails with a _Superfluous Disable Command Violation_ swiftlint errors in the two demo apps:
```
...full path omitted.../Carthage/Checkouts/SwiftyStoreKit/SwiftyStoreKit-iOS-Demo/AppDelegate.swift:46: error: Superfluous Disable Command Violation: SwiftLint rule 'for_where' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
...full path omitted.../Carthage/Checkouts/SwiftyStoreKit/SwiftyStoreKit-macOS-Demo/AppDelegate.swift:42: error: Superfluous Disable Command Violation: SwiftLint rule 'for_where' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)
```

Cartfile contains:  `github "bizz84/SwiftyStoreKit"`

**Comment by tbaranes** [comment]
Fixed in #270

**Comment by bizz84** [comment]
This is now available on version [0.10.8](https://github.com/bizz84/SwiftyStoreKit/releases/tag/0.10.8).
